src/components/input/index.tsx

```diff
@@ -59,6 +59,8 @@ interface PopupLabelProps extends InputProps {
 interface FloatingLabelProps extends InputProps {
   labelText?: string
   floatingLabel?: boolean
+  renderLabelPopup?: () => JSX.Element
```

Comment:
This looks unrelated, why is it needed?

Reply:
I got a TS error, that I need to add it on line 102
![image](https://user-images.githubusercontent.com/11503991/134346183-d5d42c57-8ea6-429e-b44a-7721ced4ffd5.png)

Reply:
it's probably because of the ternary nesting, and now we also have the required prop so that's why.

Reply:
It's because of the conditional, you're adding `renderLabalPopup` and `required` if there is `required` in other props. but the presence of `required` does not guarantee the presence of `renderLabelPopup`.

Reply:
I just looked at the code, I don't know why I added here anything, I will just remove this part,